src/Cake.NuGet/NuGetPackageInstaller.cs

```diff
@@ -175,6 +160,38 @@ public IReadOnlyCollection<IFile> Install(PackageReference package, PackageType
             return result;
         }
 
+        private static DirectoryPath GetPackagePath(DirectoryPath root, PackageReference package)
+        {
+            if (package.Parameters.ContainsKey("version"))
+            {
+                var version = package.Parameters["version"].First();
+                return root.Combine($"{package.Package}.{version}");
```

Comment:
> Yeah, but internally, NuGet CLI uses NuGetVersion for this that will have the same representation regardless of trailing zeros or not.

Ok, well folder will have "same" input parameters but different outcomes, currently we won't find the correct folder.
Also `nuget install GitReleaseManager -Version 0.6.0` will generate a lowecase folder because that's the package id, so that could fail on unix.

So almost list addin folder folders and select startswith best case first then fallback to ignore case  😉